client_plugin/drivers/vfile/dockerops/dockerops.go

```diff
@@ -220,31 +222,12 @@ func (d *DockerOps) StartSMBServer(volName string) (int, string, bool) {
 	// The Docker image to run in this service
```

Comment:
Why with this change, there is no need to pass "containerArgs"?

Reply:
No we don't need to create the shares or users since we have pre-defined shares inside the image already.

Reply:
Then when user need to create a share owned by specific user and with specific access permission, how can we set that? I don't think we can set in image in this case.

Reply:
That's right :)
This is a feature we don't support for now (and didn't support it before the change too).
We can add it later.

Reply:
My only concern here is in the original image, they are configurable in the code. But with this change, it seems hard coded in the image. Is there any way that make it configurable other than hard coded?

Reply:
Yes we will need to update the vfilesmb image to include customize functions for creating new users. We can add this when we support the user defined username functionality.